example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -734,7 +734,6 @@ describe('When there are granule differences and granule reconciliation is run',
     });
   });
 
-  // TODO: fix tests in CUMULUS-3806 when CreateReconciliationReport lambda is changed to query postgres
```

Comment:
were these tests fixed incidentally by the shift to Postgres?

Reply:
Fixed yes, but intentionally.    This comment just wasn't cleaned up in the prior PRs.